.markdownlint.json

```diff
@@ -1,6 +1,7 @@
 {
   "default": true,
   "MD013": false,
+  "MD026": { "punctuation": ".,;:。，；：" },
```

Comment:
Configuring to allow exclamation points and question marks at the end of headings